### ISSUE TITLE
Rename Space Objects to Locations

### ISSUE THREAD

**Issue by darozak** [Darwin-River/Ex-Machinis#108]
I think it will be more informative to label the Space Objects page to Locations.

**Comment by darozak** [comment]
In the process of changing http://advolition.com/objects to /locations I realized that I need to change the location links that are in all of the tables.   These links are located in the table descriptions which can be found at resources/assets/js/components.

However, as Fernando noted in a [previous issue](https://github.com/Darwin-River/Ex-Machinis/issues/71),  I need to call `npm run dev` from the root directory to compile both js and css scripts contained in the assets directory.

I actually did this by logging into root and going to var/www where a call to `npm run prod` successfully compiled the altered js script and the changes were visible online!

So... having learned how to do this, I need to implement it throughout all the tables to complete this mod.  Stay tuned!
